lib/conditionals.md

```diff
@@ -5,8 +5,10 @@ instructor_notes: Feel free to re-organize the headings (or add/remove headings)
 
 # What is a conditional?
 
-Replace me with your response...
+A conditional is a statement which sets the parameters for which parts of the code will be executed.
```

Comment:
How about, "it sets the rules for which parts of the code will be executed."

Does that work? I want to say "decision rules", but fear we will run into the same issue. :)